Makefile

```diff
@@ -108,6 +110,9 @@ runtime:
 dummy_device:
 	$(MAKE) -C runtime dummy_device
 
+oqc:
+	$(MAKE) -C frontend/catalyst/oqc/src oqc
+
```

Comment:
could you also update make clean to clean the oqc build folder?

Reply:
Added :+1: